src/main/resources/public/ts/models/teacher/ReleveNote.ts

```diff
@@ -773,6 +803,13 @@ export class ReleveNote extends  Model implements IModel {
             column.push('appreciation_matiere_periode');
         }
 
+        if (this.exportOptions.previousAverage ) {
+            _.each(this.getPreviousPeriod(), (p) => {
+                let label = lang.translate('average') + lang.translate("viescolaire.periode." + p.type).charAt(0) + p.ordre
+                header += `; ${label}`;
+               column.push(label);
+            });
```

Comment:
```suggestion
  this.getPreviousPeriod().forEach((p: any) => {
                let label: string = lang.translate('average') + lang.translate("viescolaire.periode." + p.type).charAt(0) + p.ordre
                header += `; ${label}`;
                column.push(label);
            });
```

Malheureusement pas la possibilité de typer en `Periode` car tu as des champ pas reconnu de l'objet Periode...